db/migrate/004_create_marty_roles.rb

```diff
@@ -1,4 +1,4 @@
-class CreateMartyRoles < ActiveRecord::Migration
+class CreateMartyRoles < ActiveRecord::Migration[4.2]
   def change
```

Comment:
the migrations in marty have 4.2 specified.  but in your mcfly fork, it is 5.1.. is this intentional?

Reply:
Nope

Reply:
should these marty migrations also be specifying 5.1 then?

Reply:
Yeah, we can make it 5.1

Reply:
@jockc I checked it once again and i think we should keep 4.2 migration, because AR 5 works differently in some cases. So i've specified 4.2 in migrations and added 4.2 support for mcfly gem as well.

Apart from that i found out that mcfly used rails 4 in tests and fixed it as well.